src/renderer/editor/clipboard.ts

```diff
@@ -46,7 +46,12 @@ export const pasteText: () => void = () => {
   const { clipboard } = store.getState();
 
   // Paste after the last word in the selection
-  const { endIndex } = range;
+  const { startIndex, endIndex } = range;
+
+  if (clipboard.length && endIndex - startIndex > 1) {
```

Comment:
`if (clipboard.length > 0 && getLengthOfRange(range) > 1) {`

Reply:
Split into different sections, where they are each more relevant.

Don't initiate paste if you have nothing in your clipboard:
https://github.com/chloebrett/mlvet/blob/4fb7ad2dc3f83e9f97a2d7fc4f22df4cfda59a76/src/renderer/editor/clipboard.ts#L52-L54

and only perform a deleting paste if your selection is more than a word:
https://github.com/chloebrett/mlvet/blob/4fb7ad2dc3f83e9f97a2d7fc4f22df4cfda59a76/src/renderer/store/transcriptionWords/reducer.ts#L81-L88